### DISCUSSION TITLE
VLC via Telnet...Possible as a media player for MA?

### DISCUSSION THREAD

**Discussion by mkono87** [music-assistant/support#1301] [Q&A] [answered]
### What version of Music Assistant has the issue?

2023.06.1

### The problem

Wondering if its possible to get VLC via Telnet as a media play for MA? I have a PI audio hat with this rpi so I was hoping I can just feed the music through it. If not is there something else I can use?

### How to reproduce

Not really an issue.

### What version of Home Assistant Core are your running

2023.06.1

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
You can use picoreplayer

**Comment by erkr** [comment]
The music assistant integration is still in beta. Current version exposes MA players to HA. The production release will als support to use HA media players in MA, given these players met the requirements for MA. That will include the VlC player

**Comment by OzGav** [comment]
Also this is not an issue. Moving to discussions

**Comment by Surgical17** [comment]
Upvoting this. I am running VLC on HA on a laptop. 
So MA makes it handy to play Spotify or Radio or anything via the laptop speakers. Its the reason I got V1 of MA.

**Reply by OzGav** [reply]
You can use the squeeze lite addon instead to solve this.

**Reply by Surgical17** [reply]
Link?

**Reply by OzGav** [reply, marked as answer]
https://github.com/large/hassio-addons/tree/master/squeezelite
